src/Connection/Http1Connection.php

```diff
@@ -374,24 +382,28 @@ private function readResponse(
                             // to resolve promise with headers.
                             $chunk = null;
 
-                            /** @psalm-suppress PossiblyNullReference */
-                            do {
-                                /** @noinspection CallableParameterUseCaseInTypeContextInspection */
-                                $parser->parse($chunk);
-                                /** @noinspection NotOptimalIfConditionsInspection */
-                                if ($parser->isComplete()) {
-                                    break;
-                                }
-
-                                if (!$backpressure instanceof Success) {
-                                    yield $this->withCancellation($backpressure, $bodyCancellationToken);
-                                }
-
-                                /** @psalm-suppress TypeDoesNotContainNull */
-                                if ($this->socket === null) {
-                                    throw new SocketException('Socket closed prior to response completion');
```

Comment:
Seems like this also got accidentally removed due to rebasing?

Reply:
Yep, missed adding that back. Fixed.